patches/components-segmentation_platform-embedder-BUILD.gn.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/components/segmentation_platform/embedder/BUILD.gn b/components/segmentation_platform/embedder/BUILD.gn
+index 385ab7e469f4e0c02500295edc7f24cedaf0ac6b..9bbac1528bcf0c7c1589c4962bee0ff9c5febaf7 100644
+--- a/components/segmentation_platform/embedder/BUILD.gn
++++ b/components/segmentation_platform/embedder/BUILD.gn
+@@ -2,6 +2,8 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//components/optimization_guide/features.gni")
++
+ if (is_android) {
+   import("//build/config/android/config.gni")
+   import("//build/config/android/rules.gni")
+@@ -21,11 +23,14 @@ source_set("embedder") {
+     "//components/optimization_guide/core",
+     "//components/optimization_guide/core:features",
+     "//components/segmentation_platform/internal",
+-    "//components/segmentation_platform/internal:optimization_guide_segmentation_handler",
+     "//components/segmentation_platform/internal/proto",
+     "//components/segmentation_platform/public",
+     "//url",
+   ]
++
++  if (build_with_tflite_lib) {
```

Comment:
I think you can do a single-line patch here:
```
import("//components/optimization_guide/features.gni") if (!build_with_tflite_lib) { deps -= [ ...] }
```

Reply:
Thank you Aleksey. I simply included an [upstream change](https://github.com/chromium/chromium/commit/9352250ae5e558) from Cr107 here. If we merge before Brave is based on Cr107, I'll turn it into a one-liner as you suggested. Otherwise, that `.patch` file will simply be removed.

Reply:
We can also wait, 107 should go in soon.

Reply:
Removed.